ODPi-Runtime.md

```diff
@@ -273,7 +473,7 @@ Best practices for compatible apps to be portable and operator friendly:
 
 -   Applications SHOULD NOT assume that an Oracle JRE is being used.
 
--   Applciations SHOULD include both Microsoft batch or PowerShell as well as Unix-compatible shell scripts.
+-   Applications SHOULD include both Microsoft batch or PowerShell as well as Unix-compatible shell scripts.
```

Comment:
At this point it just makes sense to review this overall. Some of the updates (like this typo fix of  Applciations) are included since we noticed them while reviewing HCFS bits. I don't think it is worth our time to separate them in a stand-alone pull request. Of course, if you see something that clearly doesn't belong -- please flag that.